BootloaderCorePkg/Tools/SignUtility.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python

```

Comment:
why do we need SignUtiltiy.py and SingleSign.py ?  Can we use just one file to handle that ?  
I think the higher level functions in SignUtiltiy.py can be kept inside CommonUtility.

Reply:
Functionality in SingleSign.py import some functionality from CommonUtility.
Keeping higher level functions in SignUtiltiy.py to CommonUtility (Which exists now) creates cyclic dependency between modules. Thought of keeping as a separate file.

Reply:
OK